### PR TITLE
add the header and the first card

### PR DESCRIPTION
**Pull request by Sascha-Pionczewski**
i have add the heading and the first card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,27 @@
...
+      <h1>Quiz App</h1>
+    </header>
+    <main>
+      <article class="quiz-card">
+        <div class="bookmark_icon">
+          <img src="img/primary-bookmarks-list-add.svg" alt="#" />
+        </div>
+        <h2>How many keys does a piano have?</h2>
+        <button type="button" class="button_show_answer">Show Answer</button>
+        <!-- the answer is 88 keys -->
```
Comment (mpagels) [inline index.html:22]: This is nice to know, but unfortunately not the best practice to put this comment here ;-)

File: index.html [hunk 2]
```diff
@@ -8,6 +8,27 @@
...
+        <div>
+          <ul class="categories_list">
+            <li class="categories_list_item">#html</li>
+            <li class="categories_list_item">#flexbox</li>
+            <li class="categories_list_item">#css</li>
+          </ul>
```
Comment (mpagels) [inline index.html:28]: Just a personal note, I would add here `hashtag` names that are correct to the question. ;-)

File: index.html [hunk 3]
```diff
@@ -8,6 +8,27 @@
...
+        </div>
```
Comment (mpagels) [inline index.html:29]: Do you need the `div` here?

File: styles.css [hunk 4]
```diff
@@ -4,6 +4,74 @@
...
+}
+
+.quiz-card {
+  background-color: beige;
+  margin: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-content: center;
+  padding: 10vh;
```
Comment (mpagels) [inline styles.css:39]: Using `vh` could be a problem, if the view height from the view port is changing. Would recommend using `em` here.

File: styles.css [hunk 5]
```diff
@@ -4,6 +4,74 @@
...
+  position: relative;
+  justify-content: space-around;
+  height: 40vh;
```
Comment (mpagels) [inline styles.css:42]: See my other comment about the `em` part ;-)

File: styles.css [hunk 6]
```diff
@@ -4,6 +4,74 @@
...
+  color: white;
+  background-color: orangered;
+}
+
+.categories_list {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+  list-style: none;
+
```
Comment (mpagels) [inline styles.css:62]: Just a personal note, but here is a just an empty line. You could delete this for beauty reason ;-)

File: styles.css [hunk 7]
```diff
@@ -4,6 +4,74 @@
...
+  left: 30px;
+  bottom: 15px;
+}
+
+.categories_list_item {
+  background-color: gray;
+  color: #f1a601;
+  padding: 5px 10px 5px 10px;
```
Comment (mpagels) [inline styles.css:70]: If you write it this way, you can just use `padding: 5px 10px`. Try it out ;-)

File: styles.css [hunk 8]
```diff
@@ -13,6 +81,7 @@
   height: 10vh;
   width: 100vw;
   justify-content: space-around;
+  padding: 1vh;
```
Comment (mpagels) [inline styles.css:84]: See my other comment about the em part ;-)

**Review by mpagels — APPROVED** [review]
Hey Sascha, looks very good. Have just some minor improvements. Please have a look.
Very good work here. Keep it up!!

**Review by lenzloesche — APPROVED** [review]
lgtm

**Pull request merged**